distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Widget_Helpers.enso

```diff
@@ -21,11 +77,3 @@ parse_type_selector =
     options = names.zip choice . map pair-> Option pair.first pair.second
     Single_Choice display=Display.Always values=options
 
-## PRIVATE
-   Selector for type argument on `Column.parse`.
-join_kind_selector : Single_Choice
-join_kind_selector =
-    choice = ['Join_Kind.Inner','Join_Kind.Left_Outer','Join_Kind.Right_Outer','Join_Kind.Full','Join_Kind.Left_Exclusive','Join_Kind.Right_Exclusive']
-    names = ['Inner', 'Left Outer', 'Right Outer', 'Full', 'Left Exclusive', 'Right Exclusive']
-    options = names.zip choice . map pair-> Option pair.first pair.second
-    Single_Choice display=Display.Always values=options
```

Comment:
I assume this is removed because the default widget works correctly now?

Reply:
yes - this one isn't adding anything.